config/galaxy.yml.sample

```diff
@@ -1358,9 +1358,6 @@ galaxy:
   # of Galaxy's stable API.
   #enable_beta_workflow_modules: false
 
-  # Enable import and export of workflows as Galaxy Format 2 workflows.
-  #enable_beta_workflow_format: false
-
```

Comment:
Yeah - this should be rolled back also - must have been generated against dev.

Reply:
@jmchilton yep 😖

Reply:
In fact, it seems that a `make config-rebuild` was missed in https://github.com/galaxyproject/galaxy/pull/6776